src/Parser/DemonHunter/Vengeance/Modules/Spells/SigilOfFlame.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+
+import { formatPercentage, formatThousands, formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class SigilOfFlame extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    enemies: Enemies,
+  };
+
+  previousBuff = 0;
+  currentBuff = 0;
+  difference = 0;
+  stacked = 0;
```

Comment:
Taking into account the usages of this class property, I think its name is probably fine but its meaning is somewhat unclear. I recommend leaving a comment here to explain what it tracks. e.g. "This tracks the amount of targets hit by a cast of Sigil of Flame."